spec/form_options_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'action_view'
+require 'currency_select'
+
+module ActionView
+  module Helpers
+    describe CurrencySelectTag do
+      let(:tag) do
+        template_object = ActionView::Base.new(ActionView::LookupContext.new([]), {}, nil)
+        CurrencySelectTag.new('user', 'currency', template_object, {})
+      end
+
+      describe 'currency_select' do
+        it 'should not have a selected option' do
```

Comment:
To me, these two spec descriptions ("should (not) have a selected option") seem to contradict each other. I propose adding more context, i.e., the precondition for this.

Reply:
Done.  I've added the changelog item and expanded on the test descriptions.  Apologies for the delay.